spark/src/main/scala/geotrellis/spark/io/cog/COGValueReader.scala

```diff
@@ -102,18 +95,18 @@ trait COGValueReader[ID] {
         bands
           .zipWithIndex
           .filter { case (band, _) =>
-            band >= 0 && band < sourceMultibandTile.bandCount
+            band >= 0 && band < sourceGeoTiff.bandCount
           }
           .toArray
 
       val (targetBands, targetBandsIndexes) = targetBandsWithIndex.unzip
 
       val croppedTiles: Array[Tile] =
         try {
-          sourceMultibandTile
-            .crop(gridBounds, targetBands)
-            .bands
-            .toArray
+          sourceTile match {
```

Comment:
So this change is no longer required because the match should be happening above in `crop` method?

Reply:
Yeah, with the implicit class imported it can access the `crop` method directly. https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/crop/MultibandTileCropMethods.scala#L34-L49